docs/install-config/configure-internal-tls.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Configure Internal TLS communication between Harbor Component
+weight: 30
+---
+
+By default, The communication between Harbor's component are use http protocol which regards as insecure in production environment. From Harbor v2.0, TLS can be enabled for internal network. In production environments, always use HTTPS is a best practice.
+
+It is introduced a new item `internal_tls` in `harbor.yml` file. To enabled internal TLS is just set the `enabled` to `true` and set the `dir` value to the path of directory that place internal cert file.
+
+All the certs files can automatically generated by `prepare` tool. 
+```bash
+docker run -v /:/hostfs goharbor/prepare:v2.0 gencert -p /path/to/internal/tls/cert
+```
+
+User also can provides their own CA to generate other . Just put CA key pair on internal tls cert directory and name them as `harbor_internal_ca.key` and `harbor_internal_ca.crt`.
+
+Besides, User could provide all the certs by themselves either. But,there are some constraints in certs files:
```

Comment:
'Besides, User could provide all the certs by themselves either. But,there are some constraints in certs files:' -> 'However, there are some constraints for the certs'